app/Policies/CheckoutPolicy.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Policies;
+
+use App\Checkout;
+use App\Role;
+use App\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class CheckoutPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view the checkout.
+     */
+    public function view(User $user, Checkout $checkout)
+    {
+        return $user->hasRole(Role::COLLEGIST);
+    }
+
+    /**
+     * Determine whether the user can create and handle transactions.
+     */
+    public function handle(User $user, Checkout $checkout)
+    {
+        return $user->hasRole(Role::STUDENT_COUNCIL);
+    }
+
+    public function handleAny(User $user)
+    {
+        return $user->hasRole(Role::STUDENT_COUNCIL);
+    }
```

Comment:
What is this used for?

Reply:
Will use it with next commit, was waiting for the permission PR to get in.